.github/workflows/build_docs.yaml

```diff
@@ -71,7 +71,7 @@ jobs:
         level: error
         fail_on_error: true
     - name: Check license headers
-      run: cargo xtask check_license_headers --fix-it || echo "License information needs an update"
+      run: cargo xtask check_license_headers --fix-it
     - name: Suggest license header changes
```

Comment:
This step is now useless

Reply:
Could you explain why that is useless?

Reply:
But there's no failure.

For example:

1. I modified `internal/core/lib.rs` and removed the license and copyright headers.
2. I run `cargo xtask check_license_headers --fix-it`

and I get this:

```
$ cargo xtask check_license_headers --fix-it
    Blocking waiting for file lock on build directory
   Compiling xtask v1.2.0 (/Users/simon/src/slint/xtask)
    Finished dev [unoptimized + debuginfo] target(s) in 8.09s
     Running `target/debug/xtask check_license_headers --fix-it`
Fixing up "/Users/simon/src/slint/internal/core/lib.rs" as instructed. It's missing a license header.
All files are ok.
$ echo $?
0
$ git status
On branch simon/fix-license-header-check
Your branch is ahead of 'origin/simon/fix-license-header-check' by 1 commit.
  (use "git push" to publish your local commits)

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   internal/core/lib.rs

no changes added to commit (use "git add" and/or "git commit -a")
$ git diff
diff --git a/internal/core/lib.rs b/internal/core/lib.rs
index 2cca6e85e..f151ac24f 100644
--- a/internal/core/lib.rs
+++ b/internal/core/lib.rs
@@ -1,3 +1,6 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
+
 // cSpell: ignore sharedvector textlayout

 #![doc = include_str!("README.md")]

```

I must be missing something, or why is this useless now?

Reply:
To clarify further:

Before this PR:
- Running the xtask may fail or pass, the return code is _always_ ignored.
- If the task was able to fix some files, that diff is posted by the review bot.

After this PR:
- Running the xtask may fail because of something the xtask couldn't fix -> the CI fails (as it should IMO)
- If the xtask was able to "fix" something, the diff is posted by the review bot.

Reply:
Ok your explaination makes sense.  I thought Tobias did it like that because it would fails if there was things that needed to be fixed.